website/content/en/highlights/2021-11-16-CSV-enrichment.md

```diff
@@ -0,0 +1,68 @@
+2021-11-16-CSV-enrichment.md
```

Comment:
```suggestion
```

Or do you mean to have that there? I haven't seen other markdown files with the filename in the header.